manager/container.go

```diff
@@ -270,6 +271,30 @@ func (c *containerData) updateLoad(newLoad uint64) {
 	}
 }
 
+func (c *containerData) updateSubcontainerStats(stats *info.ContainerStats, path string) (*info.ContainerStats, error) {
+	// type []info.ContainerReference
+	subcontainers, err := c.handler.ListContainers(container.ListRecursive)
+	// glog.Infof("subcontainers for container %v are %v", c.info.Name, subcontainers)
+	if err != nil {
+		// glog.Infof("returning from subcontainerstats with error %v", err)
+		return stats, err
+	}
+	for _, sub := range subcontainers {
+		subpath := path + "/" + strings.TrimLeft(sub.Name, path)
+		loadStats, err := c.loadReader.GetCpuLoad(sub.Name, subpath)
+		if err != nil {
+			// glog.Infof("Failed to get subcontainer stats with error %v", err)
+			continue
+		}
+		stats.UpdateTaskStats(loadStats)
+		c.updateLoad(loadStats.NrRunning)
+		// convert to 'milliLoad' to avoid floats and preserve precision.
+		stats.Cpu.LoadAverage = int32(c.loadAvg * 1000)
+		// glog.Infof("Added stats from cpuloadpath %v and name %v", subpath, sub.Name)
+	}
+	return stats, nil
```

Comment:
Just tracking the time of last update and comparing might be helpful to avoid double checking.